hack/makefiles/README.md

```diff
@@ -0,0 +1,11 @@
+This directory allows extending the Makefile by adding custom
```

Comment:
`s/the Makefile/the top-level Makefile/`

(we don't have any other makefiles, but it makes sense to clarify that we're not talking about Makefile in the same directory...)